packages/block-editor/src/components/block-canvas/style.scss

```diff
@@ -2,12 +2,5 @@ iframe[name="editor-canvas"] {
 	width: 100%;
 	height: 100%;
 	display: block;
-}
-
-iframe[name="editor-canvas"]:not(.has-editor-padding) {
 	background-color: $white;
 }
-
-iframe[name="editor-canvas"].has-editor-padding {
-	padding: $grid-unit-60 $grid-unit-60 0;
```

Comment:
Removed bottom padding to match what the style editor already has:

https://github.com/WordPress/gutenberg/blob/63e4b3fccdfacf80151176fe4d359924b07dfb03/packages/edit-site/src/components/block-editor/style.scss#L40-L40